tests/ttnn/python_api_testing/non_working_unit_tests/wormhole/test_selu.py

```diff
@@ -0,0 +1,73 @@
+# SPDX-FileCopyrightText: © 2023 Tenstorrent Inc.
+# SPDX-FileCopyrightText: © 2023 Tenstorrent Inc.
+
+# SPDX-License-Identifier: Apache-2.0
+
+from loguru import logger
+import random
+import pytest
+import torch
+import ttnn
+import traceback
+
+from tests.ttnn.utils_for_testing import assert_with_pcc
+from tests.ttnn.python_api_testing.sweep_tests import ttnn_ops
+
+
+def run_eltwise_exp_tests(
+    input_shape,
+    dtype,
+    dlayout,
+    in_mem_config,
+    output_mem_config,
+    data_seed,
+    device,
+):
+    torch.manual_seed(data_seed)
+    x = torch.Tensor(size=input_shape[0]).uniform_(-100, 100).to(torch.bfloat16)
+
+    try:
+        # get ref result
+        ref_value = torch.nn.functional.selu(x)
+
+        x = ttnn_ops.setup_ttnn_tensor(x, device, dlayout[0], in_mem_config[0], dtype[0])
+
+        tt_result = ttnn.selu(x)
+        tt_result = ttnn_ops.ttnn_tensor_to_torch(tt_result, output_mem_config)
+
+    except Exception as e:
+        logger.warning(f"Test execution crashed: {e}")
+        print(traceback.format_exc())
+        raise e
+
+    assert len(tt_result.shape) == len(ref_value.shape)
+    assert tt_result.shape == ref_value.shape
+    assert_with_pcc(ref_value, tt_result, 0.99)
+
+
+test_sweep_args = [
+    (
+        [(3, 2, 192, 32)],
+        [ttnn.bfloat16],
+        [ttnn.TILE_LAYOUT],
+        [ttnn.DRAM_MEMORY_CONFIG],
+        ttnn.L1_MEMORY_CONFIG,
+        6861134,
+    ),
+    (
+        [(12, 224, 224)],
```

Comment:
Could you please add tests, where last two dimensions are not divisible by 32? Thanks in advance!